### PR TITLE
Kg 13959

### PR DESCRIPTION
**Pull request by davecombs**
This just changes to use 'copy' instead of 'concat', as the project artifacts (JS files) are used individually by gateway.client.javascript and amqp-0.9.1. They are not used as a single file, so there's no reason to concatenat them. I also tried to make the package.json and bower.json files a little more standard format.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -10,21 +10,19 @@ module.exports = function(grunt) {
...
 
     });
     
-    grunt.loadNpmTasks('grunt-contrib-concat');
+    grunt.loadNpmTasks('grunt-contrib-copy');
 	
     grunt.loadNpmTasks('grunt-contrib-clean');
 	
-    grunt.registerTask('default', [ 'clean', 'concat']);
+    grunt.registerTask('default', [ 'clean', 'copy']);
```
Comment (dpwspoon) [inline Gruntfile.js]: minor: formatting

File: bower.json [hunk 2]
```diff
@@ -1,24 +1,35 @@
 {
   "name": "gateway.client.javascript.util",
-  "version": "5.0.0",
-  "authors": [
-    "Kaazing"
-  ],
+    "version": "5.0.0",
```
Comment (dpwspoon) [inline bower.json]: minor: formatting (Wish we had a common json/js coding template for eclipse (Like we do with Java)

File: bower.json [hunk 3]
```diff
@@ -1,24 +1,35 @@
...
-    "Gruntfile.js",
-    "node_modules",
-    "bower_components",
-    "test",
-    "*.json",
-    "LICENSE",
-    "package.json",
-    "README.md"
-  ]
+
```
Comment (dpwspoon) [inline bower.json]: Do we need everything below this?  This project doesn't create/contain a bower artifact so I'm guessing not?  Example: https://github.com/angular/angular.js/blob/master/bower.json

**Comment by dpwspoon** [comment]
Looks good other then a few minor formatting things!

File: bower.json (hunk 3 above)
Comment (davecombs) [inline bower.json]: removed all this stuff. Now only includes name and devDependencies (empty for us) like angular.js

File: bower.json (hunk 2 above)
Comment (davecombs) [inline bower.json]: Removed various fields, changed to 4-space indents consistently.

**Pull request merged**